pages/eco_news_list_page.py

```
@@ -1,8 +1,87 @@
+            first_element = self.driver.find_element(By.XPATH, self.first_news_tags_list)
+            second_element = self.driver.find_element(By.XPATH, self.second_news_tags_list)
+        except NoSuchElementException:
+            print("Less than two comments found for this tag")
+            sys.exit()
+        else:
+            return first_element.text.split('|\n') + second_element.text.split('|\n')
+
+    def is_tag_in_list(self, tag):
+        list = self.get_tags_of_first_and_second_news()
+        if tag in list:
+            return 2 == operator.countOf(list, tag)
```

Ruff (0.12.2)

Yoda condition detected

Rewrite as `operator.countOf(list, tag) == 2`

(SIM300)